scripts/polkadotjs_endpoints_to_preconfigured.py

```diff
@@ -0,0 +1,248 @@
+
+            chain_data = {
+                "chainId": json_property.chainId[2:],
+                "name": json_property.name,
+                "assets": [{
+                    "assetId": 0,
+                    "symbol": json_property.symbol,
+                    "precision": json_property.precision,
+                    "icon": "https://raw.githubusercontent.com/novasamatech/nova-utils/master/icons/tokens/white/Default.svg"
+                }],
+                "nodes": [{"url": url, "name": name} for name, url in providers.items()],
+                "addressPrefix": json_property.ss58Format
+            }
+            return chain_data
+        except Exception as err:
+            # If there's a failure, print a warning and skip the connection
+            print(f"⚠️ Can't connect by {wss_url}, check if it is available? \n {err}")
+            # Do not raise the exception; instead, return None or handle it accordingly
+            return None
+
+
+def check_chain_id(chains, chain_id_to_check):
+    for chain in chains:
+        if chain.get("chainId") == chain_id_to_check:
+            return True
+    return False
+
+
+def check_node_is_present(chains_data, nodes_to_check):
+    # Iterate over each node to check
+    for node in nodes_to_check:
+        node_url = node['url']
+        found = False
+        # Iterate over each chain in the chain data
+        for chain in chains_data:
+            # Check if node URL exists in any node list of this chain
+            if any(n['url'] == node_url for n in chain['nodes']):
+                found = True
+                print(f"⚠️Node URL '{node_url}' is found in chain '{chain['name']}'.")
+                break
+        if not found:
+            return False
+    return True
+
+
+def create_json_files(pjs_networks, chains_path):
+    existing_data_in_chains = load_json_file(chains_path)
+
+    for pjs_network in pjs_networks:
+        # skip disabled networks and networks with commented providers
+        pjs_chain_name = pjs_network.get("text")
+        if '"isDisabled": "true"' in pjs_network or pjs_network.get("providers") == {}:
+            continue
+        else:
+            chain = create_chain_data(pjs_network)
+            if chain:
+                chain_name = chain.get("name")
+                print(f"Connection established for {chain_name}")
+                chain_id = chain.get("chainId")
+                # skip chains already added to config
+                is_chain_present = check_chain_id(existing_data_in_chains, chain_id)
+                # skip chains with wss already added to config, in case they have changed chain_id
+                is_node_present = check_node_is_present(existing_data_in_chains, chain.get("nodes"))
+                if is_chain_present or is_node_present:
+                    continue
+                else:
+                    add_chains_details_file(chain, chains_path)
+                    add_chain_to_chains_file(chain, chains_path)
```

Comment:
same here, if condition does not really need else here:
```suggestion
                add_chains_details_file(chain, chains_path)
                add_chain_to_chains_file(chain, chains_path)
```